src/org/parosproxy/paros/extension/manualrequest/ManualRequestEditorDialog.java

```diff
@@ -206,6 +208,20 @@ public void run() {
                 try {
                 	getMessageSender().handleSendMessage(aMessage);                    
                     postSend();
+                } catch (IOException sslEx) {

```

Comment:
Should be `SSLException` instead? (`HttpPanelSender` needs to be changed to just re-thrown the exception instead of wrapping in `IOException`.)